Sources/Rendering/OpenGL/RenderWindow/index.d.ts

```diff
@@ -66,7 +66,7 @@ export interface vtkOpenGLRenderWindow extends vtkOpenGLRenderWindowBase {
 	/**
 	 * Get the container element.
 	 */
-	getContainer(): void;
+	getContainer(): HTMLElement;
```

Comment:
The container can be null.

Reply:
Fixed

Reply:
You can use Nullable from [types.d.ts](https://github.com/Kitware/vtk-js/blob/master/Sources/types.d.ts#L4)
`Nullable<HTMLElement>`

Reply:
Great, thanks